nc-games/src/components/Reviews.jsx

```diff
@@ -22,7 +22,7 @@ return (
 <section class="cards-wrapper">
 {reviews.map((review) => { 
 return  <div class="card-grid-space" key={review.review_id}>
-        <a class="card" href={`https://boardgame-reviews.onrender.com/api/reviews/${review.review_id}`} style={{"--bg-img": `url(${review.review_img_url})`}}>
+        <a class="card" href={`/reviews/${review.review_id}`} style={{"--bg-img": `url(${review.review_img_url})`}}>
```

Comment:
Don't use `a` tags to link around React apps. They cause a page refresh which is not desirable. Switch this for a `Link` component from `react-router-dom`